library/src/main/java/org/xmtp/android/library/frames/FramesClient.kt

```diff
@@ -0,0 +1,94 @@
+package org.xmtp.android.library.frames
+
+import android.util.Base64
+import org.xmtp.android.library.Client
+import org.xmtp.android.library.XMTPException
+import org.xmtp.android.library.frames.FramesConstants.PROTOCOL_VERSION
+import org.xmtp.android.library.messages.PrivateKeyBuilder
+import org.xmtp.android.library.messages.Signature
+import org.xmtp.android.library.messages.getPublicKeyBundle
+import org.xmtp.proto.message.contents.PublicKeyOuterClass.SignedPublicKeyBundle
+import java.security.MessageDigest
+import org.xmtp.proto.message.contents.Frames.FrameActionBody
+import org.xmtp.proto.message.contents.Frames.FrameAction
+import java.util.Date
+
+class FramesClient(private val xmtpClient: Client, var proxy: OpenFramesProxy = OpenFramesProxy()) {
+
+    suspend fun signFrameAction(inputs: FrameActionInputs): FramePostPayload {
+        val opaqueConversationIdentifier = buildOpaqueIdentifier(inputs)
+        val frameUrl = inputs.frameUrl
+        val buttonIndex = inputs.buttonIndex
+        val inputText = inputs.inputText
```

Comment:
```suggestion
        val inputText = inputs.inputText ?: ""
```

Just to match iOS and remove the null check inline